packages/dmn-js-decision-table/test/spec/expression-language.dmn

```diff
@@ -23,10 +23,10 @@
       <output id="output2" label="Reason" name="reason" typeRef="string" />
       <rule id="rule1">
         <description>Bronze is really not that good</description>
-        <inputEntry id="inputEntry1" expressionLanguage="javascript">
+        <inputEntry id="inputEntry1" expressionLanguage="jruby">
```

Comment:
I changed this one because in one example `javascript` is set as default. Thus, it's no longer a non-default language.